src/components/Section.js

```diff
@@ -7,23 +7,35 @@ import { Row, Col } from 'react-bootstrap'
 
 export default class Section extends Component {
   static propTypes = {
-    actionButton: PropTypes.element
+    actionButton: PropTypes.element,
+    children: PropTypes.element,
+    name: PropTypes.string
   }
 
   render() {
     const { name, actionButton, children } = this.props
     return (
-      <div>
-        <Row className="section-header">
-          <Col sm={4}>
+      <>
+        <Row className="section-header hidden-md hidden-lg hidden-sm">
```

Comment:
I'm not sure if hiding the name row is worthwhile to the user that is looking the details of a definition